src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow weak
```

Comment:
weak allows unannotated function calls, it is often used for progressive addition of flow, that way you can start typechecking but in the absence of annotation, flow will consider the params automatically as `any`. https://flowtype.org/docs/existing.html#weak-mode